connector_magento/components/backend_adapter.py

```diff
@@ -4,12 +4,14 @@
 import socket
 import logging
 import requests
+from json.decoder import JSONDecodeError
 from urllib.parse import quote_plus
 import xmlrpc.client
 
 from odoo.addons.component.core import AbstractComponent
 from odoo.addons.queue_job.exception import RetryableJobError
 from odoo.addons.connector.exception import NetworkRetryableError
+from odoo.addons.connector_magento.exception import MagentoError
```

Comment:
I cannot find this MagentoError. Can you please point where is this imported from?

Reply:
This a new type of error introduced in connector_magento/exception.py